source/sass/buttons.scss

```diff
@@ -142,6 +142,18 @@
   }
 }
 
+button[disabled],
+.btn[disabled],
+.btn.disabled {
```

Comment:
Are we using a disabled class too? I only saw the attr. implemented.

Reply:
True. Let's keep the attribute selectors for now and add stuff in when we need it in the future. I'll remove `.btn.disabled`.